Formula/espflash.rb

```diff
@@ -0,0 +1,19 @@
+class Espflash < Formula
+  desc "Serial flasher utility for Espressif SoCs and modules based on esptool.py"
+  homepage "https://github.com/esp-rs/espflash"
+  url "https://github.com/esp-rs/espflash/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "01b65bdd7c22b422d18db6cf186cb788151f0ca0eda9533ff9478aaf7255a2cf"
+  license any_of: ["MIT", "Apache-2.0"]
+
+  depends_on "rust" => :build
+  depends_on "libuv"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/espflash --version")
+    assert_match "A command-line tool for flashing Espressif devices", shell_output("#{bin}/espflash --help")
```

Comment:
Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, adhere to the following guidelines to make sure that @chiefmikey can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.